Hunting Queries/OfficeActivity/new_sharepoint_downloads_by_UserAgent.yaml

```
@@ -0,0 +1,33 @@
+id: f2367171-1514-4c67-88ef-27434b6a1093
+name: SharePointFileOperation via devices with previously unseen user agents
+description: |
+  'Tracking via user agent is one way to differentiate between types of connecting device. 
+  In homogeneous enterprise environments the user agent associated with an attacker device may stand out as unusual.'
+requiredDataConnectors:
+  - connectorId: Office365
+    dataTypes:
+      - OfficeActivity
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1030
+query: |
+
+  let starttime = 14d;
+  let endtime = 1d;
+  let historicalActivity=
+  OfficeActivity
+  | where  RecordType == "SharePointFileOperation"
+  | where Operation in ("FileDownloaded", "FileUploaded")
+  | where TimeGenerated between(ago(starttime)..ago(endtime))
+  | summarize historicalCount=count() by UserAgent, RecordType;
```

Suggest keeping the UserId field for entity creation
```suggestion
  | summarize historicalCount=count() by UserAgent, RecordType, UserId;
```

---

As these have been tested on different data sets, I don't want to change core functionality, this could impact so we are leaving out for now.